internal/wasi/wasi.go

```diff
@@ -741,7 +741,37 @@ type SnapshotPreview1 interface {
 	// FdRenumber is the WASI function named FunctionFdRenumber
 	FdRenumber(ctx wasm.Module, fd, to uint32) wasi.Errno
 
-	// FdSeek is the WASI function named FunctionFdSeek
```

Comment:
I'll go back later and re-add the names to descriptions as I know they are missing mostly. no action needed here